### PR TITLE
Insert HTML comments

### PR DESCRIPTION
**Pull request by smfoote**

### CODE REVIEW

**Comment by smfoote** [comment]
Need to remove benchmark/bundle.js before merging.

File: test/acceptance/html.js [hunk 1]
```diff
@@ -24,6 +24,40 @@ let suite = {
...
+      expectedDom: (() => {
+        let frag = document.createDocumentFragment();
+        let comment = document.createComment(' <script>alert(\'bad stuff!\');</script> ');
+        frag.appendChild(comment);
+        return frag;
+      })()
+    },
+    {
+      description: 'HTML comment within HTML attribute',
+      template: '<div class="<!-- comment -->"></div>',
```
Comment (jimmyhchan) [inline test/acceptance/html.js:50]: FYI, this is not a comment. It's a malformed attribute.

Comment (smfoote) [inline test/acceptance/html.js:50]: Right, and I want to make sure that it is not interpreted as a comment. Maybe the test description needs to be updated?

Comment (jimmyhchan) [inline test/acceptance/html.js:50]: sorry for not being clear. Yes, test description is what i'm questioning.

Comment (smfoote) [inline test/acceptance/html.js:50]: See #96

**Pull request merged**